packages/buttons/src/elements/ButtonGroup.tsx

```diff
@@ -13,7 +13,13 @@ import { StyledButtonGroup } from '../styled';
 import { ButtonGroupContext } from '../utils/useButtonGroupContext';
 
 export interface IButtonGroupProps extends HTMLAttributes<HTMLDivElement> {
+  /** Set the selected item */
   selectedItem?: any;
+  /**
+   * Handles selection changes which applies button styling and invokes actions for the selection
```

Comment:
The grammar here is feeling pretty clunky. I think your previous description was quite a bit cleaner:

> Handles button selection changes

I was only referring to the `@param` description in my previous comment.

Reply:
Fixed in latest push.